pages/index.tsx

```diff
@@ -17,6 +17,23 @@ const Home: NextPage = () => {
   const [activeFeatureCard, setActiveFeatureCard] = useState<FeatureType>('profile');
   const [activeFeatureCode, setActiveFeatureCode] = useState({ type: 'js', info: FEATURES_CODE.profile.js });
 
+  useEffect(() => {
+    const handleVisualSize = () => {
+      const $visual = document.querySelector('.kv_bg svg') as HTMLElement;
+      if (!$visual) return;
+
+      const visualWidth = (1512 * (window.innerHeight - 64)) / 868;
```

Comment:
How about extracting magic numbers as constants and representing their meaning?
(I don't know what they mean, but here are examples.)
```
const SCREEN_MIN_WIDTH = 1512;
```